### ISSUE TITLE
[a11y] Condensed subviews should have items centered

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1476]
![image](https://user-images.githubusercontent.com/37256050/167643542-c06f941e-4628-4961-9020-f8e8628f20eb.png)

See check boxes.

Windows 10, Edge
